discovery-provider/alembic/versions/a88a8ce41f7d_balance_cache.py

```diff
@@ -0,0 +1,29 @@
+"""balance_cache
+
+Revision ID: a88a8ce41f7d
+Revises: f82eb376f471
+Create Date: 2021-01-19 16:09:39.121460
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'a88a8ce41f7d'
+down_revision = 'f82eb376f471'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table('user_balances',
+        sa.Column('user_id', sa.Integer(), nullable=False, primary_key=True),
+        sa.Column('balance', sa.String(), nullable=False),
```

Comment:
I think string is actually best here since that's how we pass balance around everywhere, but we could also use biginteger or something like that

Reply:
good callout - I figured string was the way to go since we end up returning everything to the client as a string anyways etc